### PR TITLE
[BACKLOG-12439] - Updated Xul specification and controllers to meet t…

### PR DESCRIPTION
**Pull request by wseyler**
…he new simplified specification. @bmorrise @mdamour1976

### CODE REVIEW

**Comment by wseyler** [comment]
@bmorrise @mdamour1976 Fixed problems with browse button on first two tabs.

File: kettle-plugins/mapreduce/src/main/resources/org/pentaho/big/data/kettle/plugins/mapreduce/entry/pmr/messages/messages_en_US.properties [hunk 1]
```diff
@@ -13,46 +13,46 @@ JobEntryHadoopTransJobExecutor.ModeAdvanced.Tab.JobSetup.Label=Job Setup
...
-JobEntryHadoopTransJobExecutor.ModeAdvanced.Tab.SuppressOutputKey.Label=Suppress Output of Reduce Key:
-JobEntryHadoopTransJobExecutor.ModeAdvanced.Tab.SuppressOutputValue.Label=Suppress Output of Reduce Value:
+JobEntryHadoopTransJobExecutor.MapInputStepName.Label=Input step name:
+JobEntryHadoopTransJobExecutor.MapOutputStepName.Label=Output step name:
+JobEntryHadoopTransJobExecutor.CombinerInputStepName.Label=Input step name:
+JobEntryHadoopTransJobExecutor.CombinerOutputStepName.Label=Output step name:
+JobEntryHadoopTransJobExecutor.ReduceInputStepName.Label=Input step name:
+JobEntryHadoopTransJobExecutor.ReduceOutputStepName.Label=Output step name:
+
+JobEntryHadoopTransJobExecutor.ModeAdvanced.Blocking.Label=Enable blocking:
```
Comment (bmorrise) [inline kettle-plugins/mapreduce/src/main/resources/org/pentaho/big/data/kettle/plugins/mapreduce/entry/pmr/messages/messages_en_US.properties]: Should be "Enable blocking" without the colon.

File: .project [hunk 2]
```diff
@@ -10,8 +10,14 @@
 			<arguments>
```
Comment (bmorrise) [inline .project]: This file shouldn't be checked in. Actually, as part of the mavenization tasks we are removing IDE specific files all together.

**Comment by wseyler** [comment]
Ok, I may have got it right this time.  @bmorrise @mdamour1976

**Comment by bmorrise** [comment]
I remembered some time ago that I was working on a XUL dialog that met some of the new UX specs that I think might be helpful to look at. It has some of the new padding styles as well as things like button placement and button icons. We opted not to use the built-in buttons, but create our own instead. The code can be found here:

 https://github.com/pentaho/pdi-dataservice-server-plugin/blob/master/src/main/resources/org/pentaho/di/trans/dataservice/ui/xul/dataservice-dialog.xul

**Comment by wseyler** [comment]
Used Ben's suggestion above to resolve issues with padding and with the help button not displaying an icon and it's positioning.  @bmorrise @mdamour1976

**Pull request merged**